Formula/s/scrcpy.rb

```diff
@@ -1,8 +1,8 @@
 class Scrcpy < Formula
   desc "Display and control your Android device"
   homepage "https://github.com/Genymobile/scrcpy"
-  url "https://github.com/Genymobile/scrcpy/archive/refs/tags/v3.0.tar.gz"
-  sha256 "6ad2306dcdf17a8c927691d1004ec632a694069187ded73d30113a5db780fc43"
+  url "https://github.com/Genymobile/scrcpy/archive/refs/tags/v3.0.1.tar.gz"
+  sha256 "5f40b6fa945d991cc89d764decc7f10cfa6b1b2e1072f0d5cd9e22027be28cfc"
```

Comment:
sha got changed, and now it is 64fbe801802dc4cbceed82915e1037e92351207373fbccf5340fdab2d62c958f